docs/training/apis-howto/api-core-ug.rst

```diff
@@ -301,110 +401,121 @@ considerations are:
    :meth:`~determined.core.PreemptContext.should_preempt` call is automatically synchronized across
    workers so that all workers decide to preempt or continue as a unit.
 
-#. Create a ``4_distributed.py`` training script by copying the ``3_hpsearch.py`` from
-   :ref:`core-hpsearch`.
+.. tip::
 
-#. Add launcher logic to execute one worker subprocess per slot.
+   Launchers
 
-   Start with a ``launcher_main()`` function that executes one worker subprocess per slot.
+   Typically, you do not have to write your own launcher. Determined provides launchers for Horovod,
+   torch.distributed, and DeepSpeed. For more information about launcher options, visit
+   :ref:`experiments`.
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/4_distributed.py
-      :language: python
-      :dedent:
-      :pyobject: launcher_main
+To perform distributed training with the Core API, you’ll need to use the appropriate distributed
```

Comment:
I'm unclear on what this means. Seems to be talking about order-of-operations concerns, but it's really about matching up classes and frameworks, right?

Reply:
@rb-determined-ai

Reply:
It is talking about order-of-operations concerns, basically that with torch.distributed you should initialize torch.distributed first, then call `from_torch_distributed()` afterwards.

I think rather than explicitly state the order of operations, including the docs snippet we already have:
```python
if __name__ == "__main__":
    # NEW: Initialize process group using torch.
    dist.init_process_group("nccl")

    # NEW: Initialize distributed context using from_torch_distributed
    # (obtains info such as rank, size, etc. from default torch
    # environment variables).
    distributed = det.core.DistributedContext.from_torch_distributed()
    with det.core.init(distributed=distributed) as core_context:
        main(core_context)
```
is probably sufficient.  We can say "something something this is how you do it with torch.distributed, see <docs> for other frameworks we support".

Reply:
Resolved:
The docs snippet we already have is sufficient because it already displays the order of operations. The rest of the page no longer explains the order of ops.